lib/main.js

```diff
@@ -8,6 +8,9 @@ const {Cu} = require("chrome");
 Cu.import("resource://gre/modules/ClientID.jsm");
 Cu.import("resource://gre/modules/Task.jsm");
 
+const PING_PREF = "browser.activity-streams.ping";
```

Comment:
Why have these both as constants and as prefs? We should have one or the other, not both.
I'm in favor of hardcoding this and eschewing the use of prefs entirely.